web.py

```diff
@@ -230,39 +262,13 @@ async def pairs(session=Depends(get_db)):
                 "quote_id": quote.hash,
                 "quote_name": quote.name,
                 "quote_symbol": quote.symbol,
-                "last_price": quote_price or last_price,
-                "base_volume": base_volume or 0,
-                "quote_volume": quote_volume or 0,
+                "last_price": FormattedFloat(quote_price or last_price),
+                "base_volume": FormattedFloat(base_volume or 0) or 0,
+                "quote_volume": FormattedFloat(quote_volume or 0) or 0,
```

Comment:
Are you sure you wanted it this way?

Reply:
Yes. We need first `or 0` inside brackets because `base_volume` & `quote_volume` could be None and we need the second `or 0` for formatting - `0.` in json without it